### PR TITLE
order precincts alphabetically in test deck results and printing

### PR DESCRIPTION
**Pull request by benadida**

### CODE REVIEW

File: client/src/routerPaths.ts [hunk 1]
```diff
@@ -20,10 +20,10 @@ const routerPaths = {
   }: BallotScreenProps) =>
     `/ballots/style/${ballotStyleId}/precinct/${precinctId}/language/${localeCode}`,
   printedBallotsReport: '/ballots/printed-report',
-  printTestDecks: '/ballots/print-test-deck',
-  printOneTestDeck: ({ precinctId }: PrecinctReportScreenProps) =>
-    `/ballots/print-test-deck/${precinctId}`,
   tally: '/tally',
+  printTestDecks: '/tally/print-test-deck',
+  printOneTestDeck: ({ precinctId }: PrecinctReportScreenProps) =>
+    `/tally/print-test-deck/${precinctId}`,
```
Comment (beausmith) [inline client/src/routerPaths.ts:26]: I'm concerned about these scope creep nav changes so close to code freeze.

**Review by beausmith — APPROVED** [review]
One concern noted. 

(I generally prefer thematic commits. I would be less reluctant to approve if this was simply the sort-relates changes.)

**Review by eventualbuddha — APPROVED** [review]

File: client/src/routerPaths.ts (hunk 1 above)
Comment (benadida) [reply client/src/routerPaths.ts:26]: just to be clear about this change: it was buggy before. You'd go to "tally" then click on "print test deck", and the tab would switch to "ballots".

Comment (beausmith) [reply client/src/routerPaths.ts:26]: …because the user is printing "ballots". We had never talked about this and I just don't want anything to break unintentionally. 👍

Comment (benadida) [reply client/src/routerPaths.ts:26]: Yeah, I guess then the button should probably have been under ballots to begin with?

You're right to point out that this change was stealth even if it ws not meant to be.

Thanks for highlighting and the approval nevertheless. I'll take it upon myself to test this well.
